airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/plugin.py

```diff
@@ -78,6 +82,26 @@ def pytest_generate_tests(metafunc):
             metafunc.parametrize("inputs", test_inputs)
 
 
+def pytest_collection_modifyitems(config, items):
```

Comment:
this function is quite big, I would like to avoid reading it closely to understand what it is doing, lets add a docstring explaining the purpose of it